lib/active_publisher/async/in_memory_adapter/async_queue.rb

```diff
@@ -0,0 +1,63 @@
+module ActivePublisher
+  module Async
+    module InMemoryAdapter
+      class AsyncQueue
```

Comment:
Is it redundant to call this `AsyncQueue` when it is already in the `Async` module? I wouldn't think we'd ever have a non-async queue.